corehq/apps/app_manager/views.py

```diff
@@ -1947,8 +1956,8 @@ def edit_form_actions(request, domain, app_id, module_id, form_id):
     form = app.get_module(module_id).get_form(form_id)
     form.actions = FormActions.wrap(json.loads(request.POST['actions']))
     form.requires = request.POST.get('requires', form.requires)
-    if actions_use_usercase(form.actions) and not is_usercase_enabled(domain):
-        if toggle_enabled_shortcut('user_as_a_case', domain, namespace='domain'):
+    if actions_use_usercase(form.actions) and not is_usercase_in_use(domain):
+        if toggles.USER_AS_A_CASE.enabled(domain):
```

Comment:
This is a bit confusing. The interaction between the toggle and the domain setting is quite weird.